apps/src/templates/tables/tableConstants.js

```diff
@@ -0,0 +1,49 @@
+import color from "../../util/color";
+import styleConstants from "../../styleConstants";
+
+// Constants for React tables
```

Comment:
It seems like this entire file is generally stuff for reacttabular tables. I wonder if we should rename it to reflect that, i.e. `reactabularStyles.js` or `reactabularConstants.js`?

Reply:
Although I didn't do this, I think html tables could be setup to work with these styles, so I purposefully kept it general.